plugins/mac-virtualcam/src/obs-plugin/CMakeLists.txt

```diff
@@ -26,4 +22,5 @@ set_target_properties_obs(
              LIBRARY_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/../../"
              XCODE_ATTRIBUTE_CLANG_ENABLE_OBJC_ARC YES
              XCODE_ATTRIBUTE_CLANG_WARN_SUSPICIOUS_IMPLICIT_CONVERSION YES
-             XCODE_ATTRIBUTE_GCC_WARN_SHADOW YES)
+             XCODE_ATTRIBUTE_GCC_WARN_SHADOW YES XCODE_ATTRIBUTE_CLANG_ENABLE_MODULES YES
+                                                 XCODE_ATTRIBUTE_CLANG_MODULES_AUTOLINK YES)
```

Comment:
This seems oddly formatted.

Reply:
That's because `cmake-format` can't handle custom functions with sets of tuples as variables, needs to be manually formatted.